cli/cmd/api.go

```diff
@@ -32,18 +32,17 @@ var apiHealthCmd = &cobra.Command{
 	Long:         "ping the health endpoint of happy api",
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		happyConfig, err := config.GetHappyConfigForCmd(cmd)
+		happyClient, err := makeHappyClient(cmd, sliceName, "", []string{}, false)
 		if err != nil {
 			return err
 		}
-
-		api := hapi.MakeApiClient(happyConfig)
+		api := hapi.MakeAPIClient(happyClient.HappyConfig, happyClient.AWSBackend)
```

Comment:
Can MakeAPIClient be made dependent on happyClient alone? This will simplify signatures -- also I doubt there's an intention to use happyConfig and AWSBackend coming from different sources.

Reply:
we make API clients in other contexts, for example the terraform provider where the happy Client won't exist

Reply:
I see.